saemref/supervisor.sls

```diff
@@ -7,21 +7,16 @@ supervisor:
   pkg:
     - installed
 
-supervisor_confdir:
-  file.directory:
-    - name: {{ supervisor_confdir }}
-    - require:
-      - pkg: supervisor
```

Comment:
I guess we can assume that previously called "supervisor_confdir" always exists?